packages/admin/resources/css/app.css

```diff
@@ -49,3 +49,23 @@
         top: -1px;
     }
 }
+
+@layer utilities {
```

Comment:
I think the `utilities` layer is wrong for this goal.

Reply:
I agree. Should either be in base or components.